### PR TITLE
environs 9.5.0 :snowflake:

### PR DESCRIPTION
**Pull request by ifitchet**
environs 9.5.0 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-5113]
- dev_url:        https://github.com/sloria/environs/tree/9.5.0
- conda_forge:    https://github.com/conda-forge/environs-feedstock/blob/main/recipe/meta.yaml
- pypi:           https://pypi.org/project/environs/9.5.0
- pypi inspector: https://inspector.pypi.io/project/environs/9.5.0

### Explanation of changes:

- basic build
- the tests (from the upstream repo) would require various django modules which we don't have so those tests have been skipped

[PKG-5113]: https://anaconda.atlassian.net/browse/PKG-5113?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -6,33 +6,64 @@ package:
...
+    - tests
   imports:
     - environs
+  requires:
+    - pip
+    - pytest
+    - django
+  commands:
+    - pip check
+    - pytest -v tests -k "{{ tests_to_skip }}"
```
Comment (lorepirri) [inline recipe/meta.yaml]:
```suggestion
    - pytest -v tests -k "not ({{ tests_to_skip }})"
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -6,33 +6,64 @@ package:
...
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+{% set tests_to_skip = "not (" %}
+{% set tests_to_skip = tests_to_skip + "TestDjango" %}
+{% set tests_to_skip = tests_to_skip + " or test_dj_db_url_with_deferred_validation_missing" %}
+{% set tests_to_skip = tests_to_skip + " or test_dj_db_url_with_deferred_validation_invalid" %}
+{% set tests_to_skip = tests_to_skip + " or test_dj_email_url_with_deferred_validation_missing" %}
+{% set tests_to_skip = tests_to_skip + " or test_dj_cache_url_with_deferred_validation_missing" %}
+{% set tests_to_skip = tests_to_skip + " or test_dj_cache_url_with_deferred_validation_invalid" %}
+{% set tests_to_skip = tests_to_skip + ")" %}
```
Comment (lorepirri) [inline recipe/meta.yaml]:
```suggestion
{% set tests_to_skip = "" %}
# skip tests that require Django and its plugins
{% set tests_to_skip = tests_to_skip + "test_dj_" %}
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -6,33 +6,64 @@ package:
...
 
 test:
+  source_files:
+    - tests
   imports:
     - environs
+  requires:
+    - pip
+    - pytest
+    - django
```
Comment (lorepirri) [inline recipe/meta.yaml]:
```suggestion
```

**Review by lorepirri — APPROVED** [review]
LGTM, nitpicks

**Review by lorepirri — APPROVED** [review]

File: recipe/meta.yaml [hunk 4]
```diff
@@ -6,33 +6,63 @@ package:
...
+  skip: True  # [py<36]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+{% set tests_to_skip = "" %}
+{% set tests_to_skip = tests_to_skip + "TestDjango" %}
+{% set tests_to_skip = tests_to_skip + " or test_dj_db_url_with_deferred_validation_missing" %}
+{% set tests_to_skip = tests_to_skip + " or test_dj_db_url_with_deferred_validation_invalid" %}
+{% set tests_to_skip = tests_to_skip + " or test_dj_email_url_with_deferred_validation_missing" %}
+{% set tests_to_skip = tests_to_skip + " or test_dj_cache_url_with_deferred_validation_missing" %}
+{% set tests_to_skip = tests_to_skip + " or test_dj_cache_url_with_deferred_validation_invalid" %}
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: Should it be placed near the `test` section?

**Review by skupr-anaconda — APPROVED** [review]
LGTM but placing the skip tests

File: recipe/meta.yaml (hunk 4 above)
Comment (ifitchet) [reply recipe/meta.yaml]: 😆 I hadn't noticed!

**Pull request merged**